### DISCUSSION TITLE
unable to restore purchase for webssh on ios

### DISCUSSION THREAD

**Discussion by jgalley2017** [isontheline/pro.webssh.net#1164] [Q&A]
I purchased this back in 2017 and use it sparingly but want to add a new connection and i get purchase/restore purchases prompt but restore purchases dont work. i have the apple invoice

**Comment by isontheline** [comment]
Which WebSSH version are you running on?
Are you on TestFlight?
Could you forward me your purchase receipt to team [AT} webssh (DOT] net ?
